widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java

```diff
@@ -41,6 +41,26 @@ public static Intent getIntent(
                 .putExtra(GliaWidgets.QUEUE_ID, queueId);
     }
 
+    /**
+     * Creates and fills out Intent for starting ChatActivity
+     * @param context - Context object
+     * @param contextId - Context asset ID
```

Comment:
Minor suggestion - we can add "Visitor context asset id" to be more clear about this variable and distinguish it from Android Context.

Reply:
If you agree with this suggestion, let's also edit it for the `getIntent()` function above.

Reply:
Agreed to leave it as is now.